azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java

```diff
@@ -276,6 +278,11 @@ public static void filterAdminOnlyFlowParams(final UserManager userManager,
       params.remove(FlowParameters.FLOW_PARAM_DISABLE_POD_CLEANUP);
       // Passing test version will be allowed for Azkaban ADMIN role only
       params.remove(FlowParameters.FLOW_PARAM_ALLOW_IMAGE_TEST_VERSION);
+
```

Comment:
This method is used whenever an execution is to be submitted (no matter adhoc or scheduled), see:
https://github.com/azkaban/azkaban/blob/284e8bf518239bf391a3f298fd3c881b070e2035/azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java#L263